aspnetcore/signalr/dotnet-client/sample/SignalRChatClient/MainWindow.xaml.cs

```diff
@@ -14,7 +15,18 @@ public MainWindow()
 
             connection = new HubConnectionBuilder()
                 .WithUrl("http://localhost:53353/ChatHub")
-                .Build();        
+                .Build();
+
+            #region snippet_ClosedRestart
+            connection.Closed += async (error) =>
+            {
+                // Consider waiting for some random delay 
```

Comment:
Consider moving this comment into the doc itself for 2 reasons:
1. It will then be localized.
1. Many readers ignore comments in the code.